src/sporemodder/view/syntax/XmlSyntax.java

```diff
@@ -101,7 +101,11 @@ public class XmlSyntax implements SyntaxFormatFactory {
 	public boolean isSupportedFile(File file) {
 		try {
 			// Unfortunately, this syntax highlighter is too slow for "big" files
-			return file.getName().endsWith(".xml") && Files.size(file.toPath()) < 10240;
+			return (
+				file.getName().endsWith(".xml") |
+				file.getName().endsWith(".eapdPixie") &
+				Files.size(file.toPath()) < 10240
+			);
```

Comment:
The condition should use boolean operators (`||`, `&&`), not binary ones (`|`, `&`). Also, the name conditions should be in parenthesis. Like this:
```java
(file.getName().endsWith(".xml") || file.getName().endsWith(".eapdPixie")) && Files.size(file.toPath()) < 10240
```

Reply:
Done.